matrix/matrix_operation.py

```diff
@@ -5,16 +5,20 @@
 from typing import List, Tuple
 
 
-def add(matrix_a: List[list], matrix_b: List[list]) -> List[list]:
+def add(*matrix_s: List[list]) -> List[list]:
     """
     >>> add([[1,2],[3,4]],[[2,3],[4,5]])
     [[3, 5], [7, 9]]
     >>> add([[1.2,2.4],[3,4]],[[2,3],[4,5]])
     [[3.2, 5.4], [7, 9]]
+    >>> add([[1, 2], [4, 5]], [[3, 7], [3, 4]], [[3, 5], [5, 7]])
+    [[7, 14], [12, 16]]
     """
-    if _check_not_integer(matrix_a) and _check_not_integer(matrix_b):
-        _verify_matrix_sizes(matrix_a, matrix_b)
-        return [[i + j for i, j in zip(m, n)] for m, n in zip(matrix_a, matrix_b)]
+    if all(map(_check_not_integer, matrix_s)):
```

Comment:
```suggestion
    if all(_check_not_integer(m) for m in matrix_s):
```
As discussed in CONTRIBUTING.md.